1907-11-13.xml

```diff
@@ -0,0 +1,2544 @@
+                    <p>ARABIA-PERSIAN SERVICE to Port Sudan, Djibuti, Aden, Mascat, Bersier Abbas,
+                        Lingah, Bahrein, Bushire Mohammerah, Basrah, (for Bagdad), Nov. 26th 5.5,
+                        “Sicilia” Dec, 20th 6. “Galicia” Jan. 25th S.S. "Assyria"</p>
+                    <p>Fares to Basrah : first class £26.0.0 steerage £3.00,</p>
+                    <p>
+                        <hi rend="bold">Sailings from Alexandria to Nappies and New-York.</hi></p>
+                    <p>Jan. 21 S.S "Hamburg" March 3. S.S. “Hamburg” @ret and second class fares to
+                        Naples from £10.00 £3.00 to New-York from £28.00. £20.00</p>
+                    <p>
+                        <hi rend="bold">Oriental Cruise by S.S: "Moltke"</hi></p>
+                    <p>Marth 10th from Alexandria to Jaffa, Constantinople, Greece, Sicily, Naplea,
+                        Genes and New-York. </p>
+                    <p>Apply to: HAMBURG-AMBERICA LINTE, Carmo, (Continental Hotel).<measure
+                            type="indexNo">20074-31-11-07</measure></p>
+                </div>
+
+                <!-- needs to be edited-->
+                <div type="advert" xml:id="deg-ad-nll02">
+                    <head>NORDDEUTSCHER LLOYD.</head>
+                    <p>Regular Service from ALEXANDRIA (Passenger and Freight) to
+                        NAPLES-MARSEILLES.</p>
+                    <p><name>SCHLESWIG</name> will leave ALEXANDRIA at 4 p.m. July 26, August 30,
+                        September 20, etc.</p>
+                    <p>The following steamers are intended to leave PORT-SAID:</p>
+                    <table rows="14" cols="3">
+                        <row role="label">
+                            <cell cols="6">HOMEWARD : for Bremen Hamburg via Naples, Genoa,
+                                (Gibraltar), Southampton, Antwerp.</cell>
+                        </row>
+                        <row>
+                            <cell><name>Zieten</name></cell>
+                            <cell><measure quantity="9043" unit="ton">9043</measure> Tons</cell>
+                            <cell>about <date when="1905-07-14">14 July</date></cell>
+                            <cell><name>Gneisenau</name></cell>
+                            <cell><measure quantity="8881" unit="ton">8881</measure> Tons</cell>
+                            <cell>about <date when="1905-08-14">14 August</date></cell>
+
+                        </row>
+                        <row>
+                            <cell><name>Gera</name></cell>
+                            <cell><measure quantity="5005" unit="ton">5005</measure> Tons</cell>
+                            <cell>about <date when="1905-07-31">31 July</date></cell>
+                            <cell><name>Pr. Regt. Luitpold</name></cell>
+                            <cell><measure quantity="6288" unit="ton">6288</measure> Tons</cell>
+        
```

Comment:
Every individual essays in a section should be made as item div under the section div.